lib/capybara/server/animation_disabler.rb

```diff
@@ -28,10 +28,19 @@ def insert_disable(html)
         html.sub(%r{(</head>)}, DISABLE_MARKUP + '\\1')
       end
 
+      # we allow a whitelist of selectors for which to enable css animations
+      private_class_method def self.enable_animations_for
+        if Capybara.disable_animation.is_a? String
+          Capybara.disable_animation
+        else
+          ''
+        end
+      end
+
       DISABLE_MARKUP = <<~HTML
         <script defer>(typeof jQuery !== 'undefined') && (jQuery.fx.off = true);</script>
         <style>
-          * {
+          *#{enable_animations_for} {
```

Comment:
Forcing a leading `*` restricts the CSS selectors that can be passed for no real benefit -- please remove and just set the selector if `*` if true is passed.